packages/p2p/src/components/order-details/__test__/order-details-complain-modal.spec.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { isMobile } from '@deriv/shared';
+import { act, fireEvent, render, screen, waitFor } from '@testing-library/react';
+import { requestWS } from 'Utils/websocket';
+import OrderDetailsComplainModal from '../order-details-complain-modal.jsx';
+
+const el_modal = document.createElement('div');
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isMobile: jest.fn(),
+}));
+
+jest.mock('Utils/websocket', () => ({
+    ...jest.requireActual('Utils/websocket'),
+    requestWS: jest.fn(),
+}));
+
+describe('<OrderDetailsComplainModal/>', () => {
+    beforeAll(() => {
+        el_modal.setAttribute('id', 'modal_root');
+        document.body.appendChild(el_modal);
+    });
+    afterAll(() => {
+        document.body.removeChild(el_modal);
+    });
+
+    it('should render complain modal in desktop ', () => {
+        render(<OrderDetailsComplainModal should_show_complain_modal />);
+
+        expect(screen.getByText("What's your complaint?")).toBeInTheDocument();
+    });
+
+    it('should toggle modal ', () => {
```

Comment:
@likhith-deriv IMHO test case description does not reflect the essence of the assertion.
maybe we can use:
should not show modal title if should_show_complain_modal is equal to false? or smth like this

Reply:
Ok. Thank you. I will rename it